pkg/operators/clustermanager/controllers/migrationcontroller/migration_controller.go

```diff
@@ -113,18 +112,10 @@ func (c *crdMigrationController) sync(ctx context.Context, controllerContext fac
 		return nil
 	}
 
-	// ClusterManager is deleting, we remove its related resources on hub
-	if !clusterManager.DeletionTimestamp.IsZero() {
-		return removeStorageVersionMigrations(ctx, migrationClient)
-	}
-
-	// do not apply storage version migrations until other resources are applied
-	if applied := meta.IsStatusConditionTrue(clusterManager.Status.Conditions, clusterManagerApplied); !applied {
-		controllerContext.Queue().AddAfter(clusterManagerName, 5*time.Second)
-		return nil
-	}
-
-	return applyStorageVersionMigrations(ctx, migrationClient, controllerContext.Recorder())
+	// do not need to run applyStorageVersionMigrations since all the exisiting StorageVersionMigrations should be removed
+	// in this release.
+	// remove StorageVersionMigrations related resources on hub.
+	return removeStorageVersionMigrations(ctx, migrationClient)
```

Comment:
I remember we decided to keep the existing migration CRs. In another word, no new migration CRs will be created and leave the existing ones untouched.

Reply:
we do not need to create new migration, and we do not need to remove the previous migration.

Reply:
fixed